pages/common/ipfs.md

```diff
@@ -1,7 +1,7 @@
 # ipfs
 
 > Inter Planetary File System.
-> A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
+> A peer-to-peer hypermedia protocol.
```

Comment:
I think we can include maybe a few of these adjectives, if we qualify them correctly. What about this?
```suggestion
> A peer-to-peer hypermedia protocol. Aims to make the web more open.
```